test/spark/docker-compose.yaml

```diff
@@ -65,8 +65,8 @@ services:
     environment:
       - SPARK_MODE=worker
       - SPARK_MASTER_URL=spark://spark:7077
-      - SPARK_WORKER_MEMORY=1G
-      - SPARK_WORKER_CORES=1
+      - SPARK_WORKER_MEMORY=8G
```

Comment:
Is this realistic on a GH actions runner?  I see [7 GiB](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources) for the entire runner VM.  All of the docker containers have to fit on that VM, along with a few other containers.

Reply:
You're right, it's not really realistic. I will change to 4G